libminifi/include/EventDrivenSchedulingAgent.h

```diff
@@ -38,8 +38,8 @@ class EventDrivenSchedulingAgent : public ThreadedSchedulingAgent {
   /*!
    * Create a new processor
    */
-  EventDrivenSchedulingAgent(std::shared_ptr<core::Repository> repo)
-      : ThreadedSchedulingAgent(repo) {
+  EventDrivenSchedulingAgent(const std::shared_ptr<core::Repository> &repo, const std::shared_ptr<Configure> &configure)
```

Comment:
These schedulers stop in a way in which we don't guarantee threads to stop. We wait in flow controller 1s to stop threads, thus we can't be 100% certain that the scheduled threads have stopped. They "probably" did, but we likely want to keep this pass by value to ensure that we are guaranteed shared ownership of the shared repo. Since the threads are daemon threads, they will stop eventually, but we probably want to avoid the error of a dereferencing deleted memory if we can to aid in eventual recoverability ( if that's a thing ).